app/style/Governance.less

```diff
@@ -52,16 +52,42 @@
   padding: 1em;
 }
 
+.loading-error {
```

Comment:
The governance pages are being refactored ATM to use css module instead of `.less`.
So it would be great if the styling of LoadingError component will be in it's own css module.

We have two option:
1. Create components/shared/ErrorLoading.module.css 
2. Create one css module for all shared components for now instead of css module for each component, then you could create `components/shared/shared.module.css`

After adding the module you could import it in your component and style your component - have a look at ProposalDetails.module.css on master.

Reply:
TBH I'm more fan of the second option here, since all shared components are small enough I guess one module should be enough here.
In case we see in later stage that one component styling is growing to big, we could easily then add a separate css module for it's own.

Reply:
@amassarwi I didn't see your comment before my last commits, so please ignore it. Unfortunately, I had to rebase my branch to the most recent master updates, basically my last commits are the fixes I've done so far. I have two others pull requests opened here, so probably I'll rebase it too.

Reply:
Regarding the ```.less``` modules, I totally agree. I'll push the corrections soon.